src/recs.js

```diff
@@ -62,6 +62,13 @@ export async function initTemplate(options) {
 				},
 				default: framework.template.dir,
 			},
+			{
+				type: 'list',
+				name: 'type',
+				message: 'Please select the type of recommendations:',
+				choices: ['default', 'email'],
```

Comment:
Could these choices be tied to the mapping in `framework.template` somehow? I think it may require a restructure, but would allow make things simpler to add more templates in the future.

Something like:

```
framework.template = {
  dir,
  components: {
    default, // maybe rename this one to carousel, or recommendation?
    email,
  }
}
```

Then the choices would just be `Object.keys(framework.template.components)` or similar.